.github/workflows/docker_linux_x86_64.yml

```diff
@@ -0,0 +1,79 @@
+name: Docker::Linux::x86_64
+
+# **What it does**: Builds Docker images for Linux (ubuntu-22.04) architecture x86_64 and store it as artifacts.
+# **Why we have it**: To build Docker images to be uploaded to Docker Hub.
+# **Who does it impact**: Docker images uploaded to Docker Hub provide yet another way to install and use PennyLane + Lightning. It is especially useful on HPC platforms where environments can be difficult to set up.
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+concurrency:
+  group: docker_linux_x86_64-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+
+  docker:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        pl_version: ["0.34.0"]
```

Comment:
This may also be wroth setting up a PennyLane latest_release tag, so we can avoid this becoming a thing to change each release. Can be done in a follow-up PR

Reply:
Agreed. Or have a `workflow_dispatch` `input` which takes in the version?

Reply:
Yep, sounds good to me. Maybe add a TODO here and we can come back to it.